apps/aeprimop/src/aeprimop.erl

```diff
@@ -1603,7 +1775,13 @@ contract_call_fail(Call0, Fee, S) ->
     {Account, S3} = get_account(Payer, S2),
     account_spend(Account, UsedAmount, S3).
 
+<<<<<<< HEAD
```

Comment:
Rebase error. Please be carefull when reabasing... Git tells you when a file has issues. There are multiple ways to deal with them. Personally I frequenetly use `git mergetool`